From 421027daf33d9006abea7441c11e6c61be63e3cf Mon Sep 17 00:00:00 2001
From: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
Date: Thu, 29 Aug 2019 11:12:30 +0530
Subject: [PATCH] [REVERT ME] Update target sdk version and legacy external
 storage

From API level 29, apps are not allowed to store files
in public directory using getExternalStoragePublicDirectory API.

If the app needs to use legacy storage API, it can set
the target sdk to 28 and android:requestLegacyExternalStorage
to true.

Change-Id: I223d769f0abef6188ca5d2a0ff4278fa8f5791ef
Tracked-On: OAM-85399
Signed-off-by: Jeevaka Prabu Badrappan <jeevaka.badrappan@intel.com>
---
 apps/TestingCamera/AndroidManifest.xml | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/apps/TestingCamera/AndroidManifest.xml b/apps/TestingCamera/AndroidManifest.xml
index 415364c..734dbf3 100644
--- a/apps/TestingCamera/AndroidManifest.xml
+++ b/apps/TestingCamera/AndroidManifest.xml
@@ -22,7 +22,7 @@
   <uses-feature android:name="android.hardware.camera" />
   <uses-feature android:name="android.hardware.camera.autofocus" android:required="false" />
 
-  <uses-sdk android:minSdkVersion="17" android:targetSdkVersion="10000"/>
+  <uses-sdk android:minSdkVersion="17" android:targetSdkVersion="28"/>
   <uses-feature android:name="android.hardware.camera.front" android:required="false"/>
   <uses-permission android:name="android.permission.RECORD_AUDIO"/>
   <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
@@ -30,7 +30,8 @@
   <application
        android:icon="@mipmap/launcher_testingcamera"
        android:label="@string/app_name"
-       android:theme="@android:style/Theme.Holo.NoActionBar.Fullscreen">
+       android:theme="@android:style/Theme.Holo.NoActionBar.Fullscreen"
+       android:requestLegacyExternalStorage="true">
     <activity
          android:name=".TestingCamera"
          android:label="@string/app_name"
-- 
2.17.1

